alphabase/constants/aa.py

```diff
@@ -11,12 +11,14 @@
     parse_formula,
     reset_elements,
 )
-from alphabase.yaml_utils import load_yaml
 
 # We use all 128 ASCII code to represent amino acids for flexible extensions in the future.
 # The amino acid masses are stored in 128-lengh array :py:data:`AA_ASCII_MASS`.
 # If an ASCII code is not in `AA_Formula`, the mass will be set as a large value to disable MS search.
-AA_Formula: dict = load_yaml(os.path.join(CONST_FILE_FOLDER, "amino_acid.yaml"))
+AA_Formula: pd.DataFrame = pd.read_csv(
```

Comment:
just thinking out loud: given that this is a breaking change anyway, should be take the change and rename it to
`aa_formula`? (or `_aa_formula`) if it's supposed to be private?